galaxy2cwl/get_cwl_interface.py

```diff
@@ -59,12 +59,25 @@ def process_format1_json(wf_dict):
                 output_name = 'output'
             map_output_to_in_name[step_index + '_' + output_name] = input_name
             wf_inputs[input_name] = input_details
+        elif step_details['type'] == 'parameter_input':
+            input_details = {}
+            input_details['format'] = 'boolean'
+            state = json.loads(step_details['tool_state'])
+            input_details['type'] = state['parameter_type']
```

Comment:
@ieguinoa I didn't know what to do with format, that is why I took arbitrary Boolean... But the type I already parsed. This explains the confusion .